src/screens/HomePage/HomePage.tsx

```diff
@@ -131,13 +142,23 @@ export default function HomePage() {
                 setMenuOpen={setMenuOpen}
               />
               <HomepageItems isLoading={isLoading} userRegions={userRegions} />
-              <Toggle
-                showCounties={showCounties}
-                onClickSwitch={onClickSwitch}
-              />
+              <ToggleWrapper>
+                <ButtonGroup
+                  value={locationScope}
+                  exclusive
+                  onChange={onClickSwitch}
+                >
+                  <Button value={'States'}>States</Button>
+                  <Button value={'Counties'}>Counties</Button>
+                </ButtonGroup>
+              </ToggleWrapper>
             </ColumnCentered>
 
-            <Map hideLegend hideInstructions showCounties={showCounties} />
+            <Map
+              hideLegend
+              hideInstructions
+              showCounties={locationScope === 'Counties' ? true : false}
```

Comment:
we dont need the ternary here. the condition will resolve to true or false on its own. with the enum implemented, this would just need to be `showCounties={locationScope ===  MapView.COUNTIES}`